src/components/info-tile/examples/info-tile-notification.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component, h } from '@stencil/core';
+/**
+ * Displaying a notification badge
```

Comment:
I'd avoid the word "notification" here. At least to me, the word "notification" means things like a push notification on my phone, or a banner on my computer screen.

Take this example:

![image](https://user-images.githubusercontent.com/154725/203068962-0a561f8f-1b77-4b57-9742-55a318434c89.png)

(Ignore the fact that I opened the drawer of past notifications to get a banner for the screenshot. Just pretend it popped on screen just now.)

I wouldn't say it's _wrong_ to call the badge a "notification badge", but I just think it's increasing the risk of confusion without really bringing all that much value, compared to just calling it "badge".

Reply:
```suggestion
 * Displaying a badge
```

Reply:
In my head, anything that "notifies" the user that something has happened or requires attention, is a notification.

So those on the top-right would be "notification cards"
The red badges wold be "notification badge".
Then we'd have "notification dot" (a badge without number or letters.
Notification emails
Notification sound
etc…

Anyhow, your suggestions seems to be what Material Design would like to call them https://m3.material.io/components/badges/guidelines

I'm fine either way

Reply:
Yeah… as always, naming is hard 😅

The "notification dot" example does make me think "notification badge" makes sense. How about this: I'm fine with "notification badge", if you prefer that, but don't call the example "notification". Use either "notification badge", or just "badge". Good with you?

Reply:
I think when we combine them as two words "notification badge" it is fine. I also found this naming as a common practice in development 
> 
> Starting with 8.0 (API level 26), notification badges (also known as notification dots) appear on a launcher icon when the associated app has an active notification. Users can long-press on the app icon to reveal the notifications (alongside any [app shortcuts](https://developer.android.com/guide/topics/ui/shortcuts)), as shown in figure 1.
> 
> These dots appear by default in launcher apps that support them and there's nothing your app needs to do. However, there might be situations in which you don't want the to notification dot to appear or you want to control exactly which notifications to appear there.

https://developer.android.com/develop/ui/views/notifications/badges

Reply:
Already changed everything. It's OK 👌 😄